tests/integration/API/JSON/APITest.php

```diff
@@ -285,7 +286,6 @@ function testItThrowsExceptionWhenInvalidEndpointMethodIsCalled() {
   }
 
   function _after() {
-    WPHooksHelper::releaseAllHooks();
```

Comment:
Why did you remove this?

Reply:
This call is not needed because methods `WPHooksHelper::intercept*` are not used anymore. I will delete them in the next step.